sdk/typescript/src/keypairs/ed25519/publickey.ts

```diff
@@ -41,15 +43,8 @@ export class Ed25519PublicKey {
 	/**
 	 * Checks if two Ed25519 public keys are equal
 	 */
-	equals(publicKey: Ed25519PublicKey): boolean {
-		return bytesEqual(this.toBytes(), publicKey.toBytes());
-	}
-
-	/**
-	 * Return the base-64 representation of the Ed25519 public key
-	 */
-	toBase64(): string {
-		return toB64(this.toBytes());
+	override equals(publicKey: Ed25519PublicKey): boolean {
```

Comment:
Why override this?

Reply:
I assume for better type-narrowing or something?

Reply:
Yeah, this is just so the expected publicKey is an Ed25519PublicKey rather than an generic PublicKey